src/Symfony/Component/Security/Http/EntryPoint/FormAuthenticationEntryPoint.php

```diff
@@ -1,61 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Security\Http\EntryPoint;
-
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpKernel\HttpKernelInterface;
-use Symfony\Component\Security\Core\Exception\AuthenticationException;
-use Symfony\Component\Security\Http\HttpUtils;
-
-/**
- * FormAuthenticationEntryPoint starts an authentication via a login form.
- *
- * @author Fabien Potencier <fabien@symfony.com>
- */
-class FormAuthenticationEntryPoint implements AuthenticationEntryPointInterface
```

Comment:
Entry points are now part of the authenticator. We should keep the interface, as it allows customizing the behavior of authenticators, but we should deprecate the form and HTTP basic implementations in 5.4.

Reply:
#42516